pipeline/trusted-resources/examples/1-test-taskrun.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
   name: tr-1
   namespace: tekton-trusted-resources
 spec:
+  serviceAccountName: "tekton-trusted-resources-webhook"
```

Comment:
I'd avoid including this or at least don't match the webhook SA - want to avoid confusion of whether this needs to match the webhook SA.

Reply:
sure!